pkg/build/shim.py

```diff
@@ -0,0 +1,21 @@
+
+import importlib.util as util
+import json
+import sys
+import os
+
+def run(args):
+	if len(args) != 3:
+		raise Exception("shim: expected 3 arguments, got {}".format(args))
+
+	spec = util.spec_from_file_location("mod.main", "{{ .Entrypoint }}")
+	mod = util.module_from_spec(spec)
+	spec.loader.exec_module(mod)
+
+	try:
+		mod.main(json.loads(args[2]))
+	except e:
+		raise Exception("shim: executing {{.Entrypoint}} - {}".format(e))
```

Comment:
You use `raise from` https://stackoverflow.com/questions/9157210/how-do-i-raise-the-same-exception-with-a-custom-message-in-python/29442282#29442282

```suggestion
		raise Exception("shim: executing {{.Entrypoint}}") from e
```